### PR TITLE
IA-2957: Fix Bioconcutor label plus a couple other non-functional fixes

### PR DESCRIPTION
**Pull request by rtitle**
See https://broadworkbench.atlassian.net/browse/IA-2957

This updates the `Bioconductor` label in the image drop-down to `[major.minor]` in all cases.

Also fixed an issue in the Hail images due to a missing env variable (see https://broadworkbench.atlassian.net/browse/IA-2910).

### CODE REVIEW

File: terra-jupyter-aou/Dockerfile [hunk 1]
```diff
@@ -41,6 +41,7 @@ ENV PIP_USER=false
 # this is effectively ignored. On GCE VMs, this will result in failures to
 # import the pyspark package.
 ENV PYTHONPATH $PYTHONPATH:/usr/lib/spark/python
+ENV PYSPARK_PYTHON=python3
```
Comment (rtitle) [inline terra-jupyter-aou/Dockerfile:44]: @calbach FYI - setting this EV in the aou and hail images, and I removed the custom kernelspec in the hail image.

**Comment by nturaga** [comment]
Thanks for this @rtitle

File: terra-jupyter-aou/Dockerfile (hunk 1 above)
Comment (calbach) [reply terra-jupyter-aou/Dockerfile:44]: Thanks!

**Review by Qi77Qi — APPROVED** [review]
lgtm

**Pull request merged**